django-web-app/static/js/script.js

```diff
@@ -194,3 +194,12 @@ document.addEventListener('DOMContentLoaded', (event) => {
       document.documentElement.scrollTop = 0; // For Chrome, Firefox, IE and Opera
   });
 });
+
+window.addEventListener("load", myLoad);
```

Comment:
it's a better approach to use useful function names like preLoad() or preLoadFunc()
instead of myload()